app/src/main/java/se/rebeccazadig/bokholken/login/LoginRepository.kt

```diff
@@ -26,31 +26,31 @@ class LoginRepository private constructor() /*primary constructor*/ {
         }
     }
 
-    suspend fun loginInRepo(email: String, password: String)/*input*/: AuthResult /*output*/ {
+    suspend fun loginInRepo(email: String, password: String)/*input*/: Result /*output*/ {
         Log.i("Emma", "LOGIN IN REPOSITORY email=$email password=$password")
 
         return try {
             val result = myAuth.signInWithEmailAndPassword(email, password).await()
             if (result.user != null) {
```

Comment:
this if is useless:
- if signInWithEmailAndPassword is successful then result.user will always be not null
- if signInWithEmailAndPassword fails then it will throw an exception so if-statement won't be reached
See how registerInRepo works